controller/user.js

```diff
@@ -5,13 +5,16 @@ const db = require("../models/index.js");
 const models = db.sequelize.models;
 const jsonwebtoken = require("jsonwebtoken");
 const errors = require("../utils/error");
+const bcrypt = require("bcryptjs");
+const { getHashPassword } = require("./auth.js");
 
 function buildUserJson(user) {
   return {
     id: user.id,
     username: user.username,
     full_name: user.full_name,
     school_name: user.school_name,
+    email: user.email,
```

Comment:
Mình thống nhất không return email khi get users trừ get me  
Cho nên nếu e muốn return email, my suggestion is 
function buildUserJson(user, can_show_email=False)

Reply:
rename `buildUserJson` to `formatUser`